JS/script.js

```diff
@@ -0,0 +1,78 @@
+/* eslint-disable max-classes-per-file */
+/* eslint-disable class-methods-use-this */
+let bookTitle;
+let bookAuthor;
+const library = document.querySelector('.library');
+const bookList = JSON.parse(localStorage.getItem('bookList')) || [];
+
+function getInformation() {
+  bookTitle = document.getElementById('bookTitle').value;
+  bookAuthor = document.getElementById('bookAuthor').value;
+}
+
+class Book {
+  constructor(title, author) {
+    this.title = title;
+    this.author = author;
+  }
+}
+
+class BookShelf {
+  constructor() {
+    this.head = null;
+    this.tail = null;
```

Comment:
[OPTIONAL] I think you are not using these two class properties. Please consider removing them. :eyes: :eyes: